src/views/registration/attendees/content.php

```diff
@@ -23,19 +23,25 @@
 	?>
 	<h3 class="tribe-ticket__heading"><?php echo get_the_title( $post->ID ); ?></h3>
 	<?php // go through each attendee ?>
-	<div class="tribe-ticket">
 	<?php while ( $j < $ticket['qty'] ) : ?>
```

Comment:
can we use something more descriptive than `$j` here? even just like `$count` etc